mtype.cs

```diff
@@ -124,6 +156,7 @@ public class Mtype
         public int MechStrBonus { get; set; }
         [DefaultValue("")]
         public string MechBattery { get; set; } = "";
+        [JsonConverter(typeof(DamageInstanceConverter<DamageUnit>))]
```

Comment:
Should it be `[JsonConverter(typeof(DamageInstanceConverter<DamageInstance>))]`?